db.json

```diff
@@ -79,20 +74,35 @@
       "type": "tea",
       "status": "preparing",
       "start_time": "2021-01-13T06:45:25.726243Z",
-      "end_time": "2021-01-13T06:48:25.726243Z",
+      "end_time": "2021-01-13T06:51:25.726243Z",
       "action_time": "2021-01-13T06:46:25.726243Z"
     }
   ],
   "status": [
     {
       "id": 1,
       "status": "preparing",
-      "duration": 300000
+      "duration": 240000
     },
     {
       "id": 2,
-      "status": "ready",
+      "status": "preparing",
       "duration": 300000
+    },
+    {
+      "id": 3,
+      "status": "ready",
+      "duration": 0
+    },
+    {
+      "id": 4,
+      "status": "ran_out",
+      "duration": -1
+    },
+    {
+      "id": 5,
+      "status": "out_of_order",
```

Comment:
bu alanı ALL_CAPS yapmıştım db'de ama senin testin için fark etmiyorsa kalabilir

Reply:
tmmdır, ben kodda lowerCase ile handle ediyorum 👍